utils.js

```diff
@@ -2443,7 +2443,24 @@ export const getPyMetadata = async function (pkgList, fetchDepsInfo) {
         p.author = body.info.author_email.trim();
       }
       p.description = body.info.summary;
-      p.license = findLicenseId(body.info.license);
+      p.license = [];
+      if (body.info.classifiers) {
+        for (const c of body.info.classifiers) {
+          if (c.startsWith("License :: ")) {
+            let licenseName = c.split(" :: ")[c.split(" :: ").length - 1];
```

Comment:
Can we split and slice instead of double split? Are the spacing around double colon mandatory or are there deviations?

Reply:
You mean sth like this?
```suggestion
            let licenseName = c.split("::").slice(-1)[0].trim();
```